src/Bundle/Grid/Renderer/TwigGridRenderer.php

```diff
@@ -14,35 +14,24 @@
 namespace Sylius\Bundle\ResourceBundle\Grid\Renderer;
 
 use Sylius\Bundle\ResourceBundle\Grid\Parser\OptionsParserInterface;
-use Sylius\Bundle\ResourceBundle\Grid\View\ResourceGridView;
 use Sylius\Component\Grid\Definition\Action;
 use Sylius\Component\Grid\Definition\Field;
 use Sylius\Component\Grid\Definition\Filter;
 use Sylius\Component\Grid\Renderer\GridRendererInterface;
 use Sylius\Component\Grid\View\GridViewInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
 use Twig\Environment;
-use Webmozart\Assert\Assert;
 
 final class TwigGridRenderer implements GridRendererInterface
 {
-    private GridRendererInterface $gridRenderer;
-
-    private Environment $twig;
-
-    private OptionsParserInterface $optionsParser;
-
-    private array $actionTemplates;
-
     public function __construct(
-        GridRendererInterface $gridRenderer,
-        Environment $twig,
-        OptionsParserInterface $optionsParser,
-        array $actionTemplates = [],
+        private GridRendererInterface $gridRenderer,
+        private Environment $twig,
+        private OptionsParserInterface $optionsParser,
+        private RequestStack $requestStack,
```

Comment:
Is that alright in terms of BC to add service without a BC layer? I didn't check if this is released code or something historical